src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiOptionsFormFields.js

```diff
@@ -0,0 +1,360 @@
+        label={intl.formatMessage({
+          defaultMessage: 'Productaanvraag type',
+          description: 'Objects API registration options "Productaanvraag type" label',
+        })}
+        rawDescription={intl.formatMessage({
+          defaultMessage: 'The type of ProductAanvraag',
+          description: 'Objects API registration options "Productaanvraag type" description',
+        })}
+        rawErrors={getFieldErrors(name, index, validationErrors, 'productaanvraagType')}
+        errors={buildErrorsComponent('productaanvraagType')}
+        displayLabel
+      >
+        <TextInput
+          id="root_productaanvraagType"
+          name="productaanvraagType"
+          value={productaanvraagType}
+          onChange={onFieldChange}
+        />
+      </CustomFieldTemplate>
+      <CustomFieldTemplate
+        id="root_informatieobjecttypeSubmissionReport"
+        label={intl.formatMessage({
+          defaultMessage: 'Submission report PDF informatieobjecttype',
+          description:
+            'Objects API registration options "Submission report PDF informatieobjecttype" label',
+        })}
+        rawDescription={intl.formatMessage({
+          defaultMessage:
+            'URL that points to the INFORMATIEOBJECTTYPE in the Catalogi API to be used for the submission report PDF',
+          description:
+            'Objects API registration options "Submission report PDF informatieobjecttype" description',
+        })}
+        rawErrors={getFieldErrors(
+          name,
+          index,
+          validationErrors,
+          'informatieobjecttypeSubmissionReport'
+        )}
+        errors={buildErrorsComponent('informatieobjecttypeSubmissionReport')}
+        displayLabel
+      >
+        <TextInput
+          id="root_informatieobjecttypeSubmissionReport"
+          name="informatieobjecttypeSubmissionReport"
+          value={informatieobjecttypeSubmissionReport}
+          onChange={onFieldChange}
+        />
+      </CustomFieldTemplate>
+      <CustomFieldTemplate
+        id="root_uploadSubmissionCsv"
+        label={intl.formatMessage({
+          defaultMessage: 'Upload submission CSV',
+          description: 'Objects API registration options "Upload submission CSV" label',
+        })}
+        rawDescription={intl.formatMessage({
+          defaultMessage:
+            'Indicates whether or not the submission CSV should be uploaded as a Document in Documenten API and attached to the ProductAanvraag',
+          description: 'Objects API registration options "Upload submission CSV" description',
+        })}
+        rawErrors={getFieldErrors(name, index, validationErrors, 'uploadSubmissionCsv')}
+        errors={buildErrorsComponent('uploadSubmissionCsv')}
+        displayLabel
+      >
+        <Checkbox
+          id="root_uploadSubmissionCsv"
+          name="uploadSubmissionCsv"
+          value={uploadSubmissionCsv}
+          onChange={onFieldChange}
+        />
```

Comment:
Checkbox also has a required prop `label`

Reply:
I'm a bit stuck here, moving the label from the `CustomFieldTemplate` to the `Checkbox` leads to the following:

Reply:
So we keep it as is? (with `CustomFieldTemplate`)

Reply:
yes